src/Copy/BrandCopy.elm

```diff
@@ -542,3 +542,19 @@ brandCopy key =
 
         StoryThree4ImageAlt ->
             CopyText "Illustration of a man looking relaxed and happy in a garden. He is in deep conversation with a woman who is smiling."
+
+
+relatedInfo : Int -> List Int
+relatedInfo storyId =
+    case storyId of
+        1 ->
+            [ 1, 2 ]
+
+        2 ->
+            [ 2, 3 ]
```

Comment:
Did you get from @roseb04 or shall we go with 1,2 3,4 5,6 so they all get exposure?

Reply:
We could also read the stories and pick relevant ones ourselves. :)